python/lsst/daf/butler_migrate/script/rewrite_sqlite_registry.py

```diff
@@ -175,7 +176,7 @@ def transfer_everything(source_butler: Butler, dest_butler: Butler) -> None:
     # Read all the datasets we are going to transfer, removing duplicates.
     # This set could be extremely large.  For now assume this is a small
     # to medium-sized registry.
-    source_refs = set(source_butler.registry.queryDatasets(..., collections=...))
+    source_refs = set(source_butler.registry.queryDatasets(..., collections=Ellipsis))
```

Comment:
I still don't like this. It feels very wrong to make people use `Ellipsis` in standard calls to `queryDatasets`.

Reply:
As @TallJimbo mentioned, we may be able to drop it, but I'll leave that for a new ticket. For now this is needed to silence `mypy`.